ui/qt-lib/src/Ariadne/UI/Qt/Widgets/Settings.hs

```diff
@@ -0,0 +1,336 @@
+  void $ setProperty releaseNotesButton ("styleRole" :: Text) ("rightPaneSettingsButton" :: Text)
+  addRow aboutLayout versionLabel releaseNotesButton
+  addSeparator aboutLayout
+
+
+  let license = "Sample text about license"
+
+  licenseLabel <- createTwoLinesLabel "LICENSE" license
+  licenseButton <- QPushButton.new
+  void $ setProperty licenseButton ("styleRole" :: Text) ("linkButton" :: Text)
+  addRow aboutLayout licenseLabel licenseButton
+
+  for_ [releaseNotesButton, licenseButton] $ \w ->
+    QWidget.setSizePolicyRaw w Maximum Fixed
+
+  QWidget.setLayout aboutWidget aboutLayout
+
+  let as = AboutSettings{..}
+
+  connect_ releaseNotesButton QAbstractButton.clickedSignal 
+        $ \_ -> (openBrowser "https://github.com/serokell/ariadne/blob/master/CHANGELOG.md" >> pass)
+  connect_ licenseButton QAbstractButton.clickedSignal 
+        $ \_ -> (openBrowser "https://github.com/serokell/ariadne/blob/master/COPYING.md" >> pass)  
+
+  return (aboutWidget, as)
+
+initSupportSettings :: IO (QWidget.QWidget, SupportSettings)
+initSupportSettings = do
+  supportWidget <- QWidget.new
+  supportLayout <- QVBoxLayout.new
+
+  let faqText = "If you are experiencing issues, please see the FAQ on Ariadne \
+                \website for guidance on known issues."
+  faqLabel <- createTwoLinesLabel "FREQUENTLY ASKED QUESTIONS" faqText
+  faqButton <- QPushButton.new
+  void $ setProperty faqButton ("styleRole" :: Text) ("linkButton" :: Text)
+  addRow supportLayout faqLabel faqButton
+  addSeparator supportLayout
+
+
+  let clearCacheText = "Clear Ariadne Wallet cache to force resynchronization with the blockchain."
+  clearCacheLabel <- createTwoLinesLabel "CLEAR CACHE" clearCacheText
+  clearCacheButton <- QPushButton.newWithText ("Clear cache" :: String)
+  addRow supportLayout clearCacheLabel clearCacheButton
+  addSeparator supportLayout
+
+  let reportText = "If the FAQ does not solve the issue you are experiencing,\
+                   \ please use support request."
+  reportLabel <- createTwoLinesLabel "REPORT A PROBLEM" reportText
+  reportButton <- QPushButton.newWithText ("Support request" :: String)
+  addRow supportLayout reportLabel reportButton
+  addSeparator supportLayout
+
+  let downloadLogsText = "If you want to inspect logs, you can download them here. \
+                         \Logs do not contain sensitive information, \
+                         \and it would be helpful to attach them to problem reports to help \
+                         \the team investigate the issue you are experiencing \ 
+                         \Logs can be attached automatically when using the bug report feature."
+
+  downloadLogsLabel <- createTwoLinesLabel "EXPORT LOGS" downloadLogsText
+  downloadLogsButton <- QPushButton.newWithText ("Download logs" ::  String)
+  addRow supportLayout downloadLogsLabel downloadLogsButton
+  addSeparator supportLayout
+
+
+  let enableReportsText = "Share anonymous usage and diagnostic data to help \
```

Comment:
Let's hide this button and the next one until we actually collect anything. Collecting user data is a sensitive topic and hinting we would do that can be bad PR. Maybe even remove them from the code for now.